optaplanner-examples/src/main/java/org/optaplanner/examples/pas/domain/AdmissionPart.java

```diff
@@ -1,21 +1,24 @@
 package org.optaplanner.examples.pas.domain;
 
-import org.optaplanner.examples.common.domain.AbstractPersistable;
+import org.optaplanner.examples.common.domain.AbstractPersistableJackson;
+import org.optaplanner.examples.common.persistence.jackson.JacksonUniqueIdGenerator;
 
-import com.thoughtworks.xstream.annotations.XStreamAlias;
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 
-@XStreamAlias("AdmissionPart")
-public class AdmissionPart extends AbstractPersistable {
+@JsonIdentityInfo(generator = JacksonUniqueIdGenerator.class)
```

Comment:
I don't see a class hierarchy in the model; maybe the property-based ID generator would be enough.